queries/fixes/unicode/fix_channelmembers_notifyprops.sql

```diff
@@ -1 +1 @@
-UPDATE ChannelMembers SET NotifyProps = REPLACE(NotifyProps, '\u0000', '') WHERE NotifyProps LIKE '%\u0000%';
+UPDATE ChannelMembers SET NotifyProps = REPLACE(NotifyProps, '\\u0000', '') WHERE NotifyProps LIKE '%\u0000%';
```

Comment:
What about the LIKE string `%\u0000%`? Does it not need to be escaped?

Reply:
Nope, tested it and it's able to find `\u0000` values with single backslash.